aws_rules_cloudtrail/aws_s3_bucket_deleted.py

```diff
@@ -4,13 +4,18 @@ def rule(event):
         'DeleteBucket') and not event.get('errorCode')
 
 
+def helper_strip_role_session_id(user_identity_arn):
```

Comment:
You could put this in the helper library, `aws_globals`

Reply:
Additional context offline, this doesn't work right now between rules and policies. @nhakmiller is working on an implementation to formally support this